package.json

```diff
@@ -32,47 +32,47 @@
   },
   "homepage": "https://github.com/tjoskar/ng-lazyload-image#readme",
   "peerDependencies": {
-    "@angular/core": ">=2.0.0",
-    "rxjs": ">=5.0.0"
+    "@angular/core": ">=6.0.0",
+    "rxjs": ">=6.0.0"
   },
   "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "^4.2.6",
-    "@angular/common": "^4.2.6",
-    "@angular/compiler": "^4.2.6",
-    "@angular/compiler-cli": "^4.2.6",
-    "@angular/core": "^4.2.6",
-    "@angular/platform-browser": "^4.2.6",
-    "@angular/platform-browser-dynamic": "^4.2.6",
-    "@angular/platform-server": "^4.2.6",
-    "@angular/router": "^4.2.6",
+    "@angular/animations": "^6.0.0-rc.1",
+    "@angular/common": "^6.0.0-rc.1",
+    "@angular/compiler": "^6.0.0-rc.1",
+    "@angular/compiler-cli": "^6.0.0-rc.1",
+    "@angular/core": "^6.0.0-rc.1",
+    "@angular/platform-browser": "^6.0.0-rc.1",
+    "@angular/platform-browser-dynamic": "^6.0.0-rc.1",
+    "@angular/platform-server": "^6.0.0-rc.1",
+    "@angular/router": "^6.0.0-rc.1",
     "@types/core-js": "^0.9.46",
     "@types/mocha": "^5.0.0",
-    "@types/node": "^9.4.0",
-    "awesome-typescript-loader": "^3.4.1",
+    "@types/node": "^9.6.2",
+    "awesome-typescript-loader": "^5.0.0-1",
     "copyfiles": "^2.0.0",
     "core-js": "^2.5.3",
-    "electron": "^1.7.12",
+    "electron": "^2.0.0-beta.7",
     "karma": "^2.0.0",
     "karma-electron-launcher": "^0.2.0",
-    "karma-mocha": "^1.1.1",
-    "karma-sauce-launcher": "^1.1.0",
+    "karma-mocha": "^1.3.0",
+    "karma-sauce-launcher": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^3.0.0",
+    "karma-webpack": "^4.0.0-beta.0",
     "lodash.isequal": "^4.5.0",
-    "mocha": "^5.0.0",
+    "mocha": "^5.0.5",
     "null-loader": "^0.1.1",
-    "protractor": "^5.3.0",
-    "rimraf": "^2.6.1",
-    "rxjs": "5.5.6",
-    "simple-spy": "^2.1.1",
-    "source-map-loader": "^0.2.1",
-    "typescript": "^2.4.2",
-    "wallaby-webpack": "^3.9.4",
-    "webpack": "^4.2.0",
-    "webpack-cli": "^2.0.13",
-    "webpack-dev-server": "^3.0.0",
-    "zone.js": "^0.8.19"
+    "protractor": "^5.3.1",
+    "rimraf": "^2.6.2",
+    "rxjs": "6.0.0-smoosh.2",
```

Comment:
The `smoosh` version was an April joke https://twitter.com/BenLesh/status/980336499970342913

Reply:
oops. should be fixed now.